src/components/Permissions/helpers/permissionsHelper.js

```diff
@@ -106,3 +106,13 @@ export const sortDataByDate = queryResult => {
     (a, b) => new Date(b.created_at) - new Date(a.created_at)
   )
 }
+
+export const filterRemoteRequests = (remoteRequests, slugName) => {
+  const filteredRemoteRequests = { data: [] }
+  for (let i = 0; i < remoteRequests.data.length; i++) {
+    if (remoteRequests.data[i].cozyMetadata.createdByApp === slugName) {
+      filteredRemoteRequests.data.push(remoteRequests.data[i])
+    }
+  }
```

Comment:
Here you can use `Array.filter` instead of a `for` loop. Result would be the same but the code would comply with modern conventions. Also this reduce the risk of error when manipulating indexes.

```js
filteredRemoteRequests.data = remoteRequests.data.filter(data => cozyMetadata.createdByApp === slugName)
```

And if you choose to remove the `.data` complexity as suggested in previous comment, you can just use:

```js
return remoteRequests.data.filter(data => cozyMetadata.createdByApp === slugName)
```